pom.xml

```diff
@@ -2044,6 +2049,86 @@
+      <properties>
+        <!--suppress UnresolvedMavenProperty -->
+        <jacoco.master.exec.file>${maven.multiModuleProjectDirectory}/target/jacoco.exec</jacoco.master.exec.file>
+        <sonar.jacoco.reportPaths>${jacoco.master.exec.file}</sonar.jacoco.reportPaths>
+        <sonar.host.url>https://sonarcloud.io</sonar.host.url>
+        <sonar.organization>kiegroup</sonar.organization>
+        <!--suppress UnresolvedMavenProperty -->
+        <sonar.login>${env.SONARCLOUD_TOKEN}</sonar.login>
+      </properties>
+      <build>
+        <pluginManagement>
+          <plugins>
+            <plugin>
+              <groupId>org.jacoco</groupId>
+              <artifactId>jacoco-maven-plugin</artifactId>
+              <configuration>
+                <fileSets>
+                  <fileSet>
+                    <directory>${project.build.directory}</directory>
+                    <includes>
+                      <include>*.exec</include>
+                    </includes>
+                  </fileSet>
+                </fileSets>
+                <destFile>${jacoco.master.exec.file}</destFile>
+              </configuration>
+            </plugin>
+          </plugins>
+        </pluginManagement>
+        <plugins>
+          <plugin>
+            <groupId>org.jacoco</groupId>
+            <artifactId>jacoco-maven-plugin</artifactId>
+            <executions>
+              <execution>
+                <goals>
+                  <goal>report</goal>
+                  <goal>merge</goal>
+                </goals>
+                <phase>generate-resources</phase>
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <groupId>org.sonarsource.scanner.maven</groupId>
+            <artifactId>sonar-maven-plugin</artifactId>
+            <executions>
+              <execution>
+                <goals>
+                  <goal>sonar</goal>
+                </goals>
+                <phase>generate-resources</phase>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+    <profile>
+      <id>sonarcloud-analysis-pull-request</id>
+      <activation>
+        <property>
+          <name>env.ghprbPullId</name>
+        </property>
+      </activation>
+      <properties>
+        <sonar.pullrequest.provider>GitHub</sonar.pullrequest.provider>
+        <!--suppress UnresolvedMavenProperty -->
+        <sonar.pullrequest.branch>${env.ghprbSourceBranch}</sonar.pullrequest.branch>
+        <!--suppress UnresolvedMavenProperty -->
+        <sonar.pullrequest.key>${env.ghprbPullId}</sonar.pullrequest.key>
+        <!--suppress UnresolvedMavenProperty -->
+        <sonar.pullrequest.base>${env.ghprbTargetBranch}</sonar.pullrequest.base>
+        <!--suppress UnresolvedMavenProperty -->
+        <sonar.pullrequest.github.repository>${env.ghprbAuthorRepoGitUrl}</sonar.pullrequest.github.repository>
```

Comment:
https://sonarcloud.io/documentation/integrations/github/ says:
> sonar.pullrequest.github.repository=my-company/my-repo

So after looking at Jenkins properties, I'd say it should be `${env.ghprbGhRepository}`:
![Screen Shot 2019-07-11 at 13 00 29](https://user-images.githubusercontent.com/673386/61045839-ef54fb00-a3db-11e9-987a-2b4c9bc020d2.png)

Wdyt?

Reply:
Interestingly, here https://sonarcloud.io/documentation/analysis/pull-request/ they don't mention the property at all.

The analysis works - before moving this profile to kie-parent, the same environment variable was passed to sonar scanner via a property:
https://github.com/kiegroup/kie-jenkins-scripts/pull/492/files#diff-62de59f2b1d8b83097e60b2792cb1599L261

If I take a look at vehicle routing, there is no such property and the analysis works too. 

I think it's not required for newly added projects, that are paired with corresponding github repositories.

Reply:
Yes, it's probably redundant for our projects. Shall we remove it?